README.md

```diff
@@ -1,6 +1,6 @@
 <img alt="Banner" src="docs/images/banner.jpg"/>
 
-This monorepo contains the reference implementation of the [Wormhole protocol](https://wormholenetwork.com).
+This monorepo contains the reference implementation of the [Wormhole protocol](https://wormhole.com).
```

Comment:
Note for other reviewers, I very carefully ensured this url is the valid one and is not some utf magic that looks like wormhole.com and goes to an invalid website.